src/shared/components/hero-bg.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+import React, { ReactNode } from "react";
+
+interface HeroBgProps {
+  children?: ReactNode;
+}
+
+const HeroBg: React.FC<HeroBgProps> = ({ children }) => {
```

Comment:
I think it's redundant use a separate component for background, lets move him to hero.tsx